docs/source/public_server.rst

```diff
@@ -81,53 +127,68 @@ Running a public notebook server
 --------------------------------
 
 If you want to access your notebook server remotely via a web browser,
-you can do the following.  
+you can do so by running a public notebook server. For optimal security
+when running a public notebook server, you should first secure the
+server with a password and SSL/HTTPS as described in
+:ref:`notebook_server_security`.
+
+Start by creating a certificate file and a hashed password, as explained in
+:ref:`notebook_server_security`.
 
-Start by creating a certificate file and a hashed password, as explained 
-above.  Then, if you don't already have one, create a config file for the 
-notebook using the following command line::
+If you don't already have one, create a
+config file for the notebook using the following command line::
 
   $ jupyter notebook --generate-config
 
-In the ``~/.jupyter`` directory, edit the notebook config file, 
-``jupyter_notebook_config.py``.  By default, the file has all fields 
-commented; the minimum set you need to uncomment and edit is the following::
+In the ``~/.jupyter`` directory, edit the notebook config file,
+``jupyter_notebook_config.py``.  By default, the notebook config file has
+all fields commented out. The minimum set of configuration options that
+you should to uncomment and edit in :file:``jupyter_notebook_config.py`` is the
+following::
 
+     # Notebook configuration for public notebook server
      c = get_config()
 
-     # Notebook config
+     # Set options for certfile, ip, password, and toggle off browser auto-opening
      c.NotebookApp.certfile = u'/absolute/path/to/your/certificate/mycert.pem'
-     c.NotebookApp.ip = '*'
+     c.NotebookApp.ip = u'*'   # where * is the desired ip address
```

Comment:
Hum, `'*'` is actually `'*'` it has the special meaning "listen on all IPs". You can also replace it by a specific IP, if you like but in the case of a public server you want `*`